test/integration/noderesourcetopology_test.go

```diff
@@ -555,6 +555,111 @@ func TestTopologyMatchPlugin(t *testing.T) {
 			},
 			expectedNodes: []string{"fake-node-1", "fake-node-2"},
 		},
+		{
+			name: "SingleNUMANodePodLevel: Filtering out nodes that cannot fit resources in case of Guaranteed pod with multi containers",
```

Comment:
same for the unit test: let's enumerate the flows and let's check which of them we want to cover